main.py

```diff
@@ -1,7 +1,58 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    Genre.objects.create(
+        name="Western"
+    )
+    action = Genre.objects.create(
+        name="Action"
+    )
+    dramma = Genre.objects.create(
+        name="Dramma"
+    )
+
+    Actor.objects.create(
+        first_name="George", last_name="Klooney"
```

Comment:
There is a typo in the last name 'Klooney'. It should be 'Clooney' to match the update on line 43.